docs/prerequisites.md

```diff
@@ -132,28 +127,42 @@ If you prefer not to use `ccoctl`, or your system doesn't support it, the AWS CL
 4. Attach the operator's permission policy to the role:
 
     ```bash
-    curl -o albo-operator-permission-policy.json https://raw.githubusercontent.com/alebedev87/aws-load-balancer-operator/aws-cli-commands-for-sts/hack/operator-permission-policy.json
+    curl -o albo-operator-permission-policy.json https://raw.githubusercontent.com/openshift/aws-load-balancer-operator/main/hack/operator-permission-policy.json
     aws iam put-role-policy --role-name albo-operator --policy-name perms-policy-albo-operator --policy-document file://albo-operator-permission-policy.json
     ```
 
-5. Generate the operator's aws credentials:
+4. Install the operator using the OpenShift OperatorHub web UI or by running the following commands:
 
     ```bash
-    cat <<EOF > albo-operator-aws-credentials.cfg
-    [default]
-    sts_regional_endpoints = regional
-    role_arn = ${OPERATOR_ROLE_ARN}
-    web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
+    $ oc create namespace aws-load-balancer-operator
+
+    $ cat <<EOF | oc apply -f -
+    apiVersion: operators.coreos.com/v1
+    kind: OperatorGroup
+    metadata:
+      name: aws-load-balancer-operator
+      namespace: aws-load-balancer-operator
+    spec:
+      targetNamespaces: []
     EOF
-    ```
-    **Note**: mind the format of the credentials file, examples can be found in [OCP documentation](https://docs.openshift.com/container-platform/4.13/authentication/managing_cloud_provider_credentials/cco-mode-sts.html#sts-mode-about_cco-mode-sts).
-
-6. Create the operator's credentials secret with the generated aws credentials:
 
-    ```bash
-    oc -n aws-load-balancer-operator create secret generic aws-load-balancer-operator --from-file=credentials=albo-operator-aws-credentials.cfg
+    $ cat <<EOF | oc apply -f -
+    apiVersion: operators.coreos.com/v1alpha1
+    kind: Subscription
+    metadata:
+      name: aws-load-balancer-operator
+      namespace: aws-load-balancer-operator
+    spec:
+      channel: stable-v1
+      name: aws-load-balancer-operator
+      source: redhat-operators
+      sourceNamespace: openshift-marketplace
+      config:
+        env:
+        - name: ROLEARN
+          value: "${OPERATOR_ROLE_ARN}"
```

Comment:
maybe should update the value as 
```
value: "${ROLEARN}"
```
And is this ENV required on STS cluster only ?

Reply:
Updated to `$ROLEARN` to be consistent with `ccoctl` option above.